src/kaltura-player.js

```diff
@@ -57,6 +57,7 @@ class KalturaPlayer extends FakeEventTarget {
   _appPluginConfig: KPPluginsConfigObject = {};
   _viewabilityManager: ViewabilityManager;
   _playbackStart: boolean;
+  _adIsPlaying: boolean = false;
```

Comment:
I think it's better to handle it on adsController and expose getter for isPlaying

Reply:
Good idea. Will do